crates/pkgcraft-tools/src/subcmds/pkg/source.rs

```diff
@@ -463,9 +470,11 @@ fn source(targets: PkgTargets, cmd: &Command) -> anyhow::Result<ExitCode> {
     let stdout = io::stdout().lock();
     let mut out = cmd.format.make_output(stdout);
 
-    for result in pkgs.par_map(func).jobs(cmd.jobs) {
-        match result {
-            Ok(value) => {
+    let result: pkgcraft::Result<Vec<_>> = pkgs.par_bridge().map(func).collect();
+
+    match result {
+        Ok(vals) => {
+            for value in vals {
                 if cmd.bound.iter().all(|b| b.matches(&value.elapsed)) {
                     if let Some(values) = sorted.as_mut() {
                         values.push(value);
```

Comment:
Here it is collecting already, I can convert it to a try_for_each

Reply:
Again, sorting isn't a default feature.

Reply:
I'll do in a separate patch then :)